scripts/sct_deepseg_lesion.py

```diff
@@ -339,10 +339,13 @@ def main():
         sct.log.error('Please use the flag -file_centerline to indicate the centerline filename.')
         sys.exit(1)
     
-    manual_centerline_fname = arguments["-file_centerline"]
-    if "-file_centerline" in args and ctr_algo != 'manual':
-        sct.log.error('Please add to the command "-centerline manual".')
-        sys.exit(1)
+    if "-file_centerline" in args:
+        manual_centerline_fname = arguments["-file_centerline"]
+        if ctr_algo != 'manual':
```

Comment:
alternatively we can automatically set `ctr_algo='manual'` (or ignore that field) if user enters a manual centerline. It will make lighter syntax.

Reply:
Good suggestion, thank you. Done in the last commit.